web/pgadmin/utils/driver/psycopg3/server_manager.py

```diff
@@ -675,6 +675,12 @@ def create_connection_string(self, database, user, password=None):
                     with_complete_path = True
                     value = get_complete_file_path(value)
 
+                if key == 'sslrootcert':
+                    if value == 'system':
```

Comment:
can be reduced to 1 if condition `if value != 'system'` to avoid sonarqube issues.